pcsx2/IopMem.cpp

```diff
@@ -490,3 +490,14 @@ void __fastcall iopMemWrite32(u32 mem, u32 value)
 		}
 	}
 }
+
+std::string iopMemReadString(u32 mem, int maxlen)
```

Comment:
To be clear, it is in fact entirely permissible for module names, the first application of this, to not be NUL terminated in both export and import tables.